sharepoint-adfs/dsc/ConfigureFEVM.ps1

```diff
@@ -243,9 +243,14 @@ configuration ConfigureFEVM
         #**********************************************************
         # Install applications using Chocolatey
         #**********************************************************
-        cChocoInstaller InstallChoco
-        {
-            InstallDir = "C:\Program Files\Choco"
+        # cChocoInstaller InstallChoco
+        # {
+        #     InstallDir = "C:\Program Files\Choco"
+        # }
+        # Workaround for issue https://github.com/chocolatey/cChoco/issues/151
+        cChocoinstaller InstallChoco {
+            InstallDir            = "C:\Choco"
+            ChocoInstallScriptUrl = "https://github.com/Azure/azure-quickstart-templates/blob/master/sharepoint-adfs/dsc/InstallChocolatey.ps1"
```

Comment:
to do it this way you'll have to pass the location into the script... can't hardcode it.

Reply:
DSC module cChoco was updated just now with the fix for the issue, Let me use it to do a proper fix. 
I'll push the change once I tested it, hopefully today

Reply:
Now the issue is properly fixed, using the newer version of the DSC module cChoco